.drone.yml

```diff
@@ -571,6 +571,16 @@ workspace:
   base: /go
   path: src/code.gitea.io/gitea
 
+depends_on:
+  - testing
+  - translations
```

Comment:
This shouldn't depend on translations because it won't run on PRs

Reply:
@techknowlogick good catch because it would also haven't work on tag on release branch.

Reply:
I will make an other PR to remove it also for release-version

Reply:
Done in #7496

Reply:
IIRC if a depending step was not run, drone just ignores that dependency. So here in our case, if `translations` did not run because we're in a pr, the step will be run when `testing` is done.